base/docs/basedocs.jl

```diff
@@ -501,6 +501,22 @@ keywords[:ccall] = doc"""
   in a call to `convert(ArgumentType, ArgumentValue)`.
   """
 
+keywords[:llvmcall] = doc"""
+      llvmcall(IR::String, ReturnType, (ArgumentType1, ...), ArgumentValue1, ...)
+
+   Call LLVM IR string in the first argument. Similar to an LLVM function ``define``
```

Comment:
default is markdown, so should be single backticks? though I think double backticks in markdown might be harmless